tests/spec/fs.read.spec.js

```diff
@@ -87,4 +87,17 @@ describe('fs.read', function() {
       });
     });
   });
-});
+
+  it('should fail to read a file that does not exist', function(done) {
+    var buffer = new Filer.Buffer(0);
+    var fs = util.fs();
+
+    var fd = 0;
+
+    fs.read(fd, buffer, 0, buffer.length, undefined, function(error, result) {
```

Comment:
instead of `position = undefined`, i would model this test case more closely after the test for [reading from a file](https://github.com/filerjs/filer/blob/fd3de6be2c1c19421d8c83e6791cd4b19afb84c7/tests/spec/fs.read.spec.js#L22). we don't want to change too many variables at a time in our tests, because each test should only test one thing. in this case, we're testing what happens when `fd` is zero _and_ `position` is `undefined`, instead of just one or the other.